iso/Dockerfile

```diff
@@ -36,7 +36,8 @@ ADD conntrack $ROOTFS/usr/bin
 # Add cifs-utils and sshfs-fuse to mount fileshares
 ENV HOST_FOLDER_DEPS cifs-utils \
               samba-libs \
-              sshfs-fuse
+              sshfs-fuse \
+              nfs-utils
```

Comment:
Will try to find where to file an issue, as the `mount.nfs` binary does not behave as expected on requesting the version with `-V` (it returns the help text, like `-h` and possibly returns an error code)